app/src/main/res/layout-v21/appview_poa_layout.xml

```diff
@@ -93,6 +93,7 @@
           android:layout_height="wrap_content"
           android:background="#f7f7f7"
           android:visibility="gone"
+          tools:visibility="visible"
```

Comment:
Can you center the message? Seems like a top margin is being used instead

Reply:
fixed